aptos-move/block-executor/src/errors.rs

```diff
@@ -17,13 +17,24 @@ pub enum IntentionalFallbackToSequential {
     // on writes. WriteSetPayload::Direct cannot be processed to do so, as we get outputs directly.
     // We communicate to the executor to retry with capability disabled.
     DirectWriteSetTransaction,
+    /// We defensively check certain resource group related invariant violations when committing outputs
```

Comment:
Again, confusing long comment. Why is some algorithm logic described here? If the logic even slightly changes, it's more or less guaranteed that this comment will not be updated!

Reply:
removed a lot of detail.